server/controllers/logout.js

```diff
@@ -0,0 +1,5 @@
+const logout = (req, res) => {
+  res.clearCookie('token', 'username');
+};
+
+module.exports = logout;
```

Comment:
After clear a cookie, we send a message to the front like (logged out successfully) to handle the request in the front 
```js
res.clearCookie('token','username').json("msg":"signed out succefully")
```